boards/peregrine/sam4l_wm400_cape/doc/index.rst

```diff
@@ -0,0 +1,148 @@
+==================
+
+The sam4l_wm400_cape board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| HWINFO    | on-chip    | Unique 120 bit serial number        |
++-----------+------------+-------------------------------------+
+| RADIO     | on-chip    | ieee802154                          |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| TRNG      | on-chip    | entropy                             |
++-----------+------------+-------------------------------------+
+| TWIM      | on-chip    | i2c master port-interrupt           |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:zephyr_file:`boards/others/sam4l_wm400_cape/sam4l_wm400_cape_defconfig`.
+
+Connections and IOs
+===================
+
+The `SAM4L WM-400 Cape Information`_ has detailed information about board
+connections. Download the `SAM4L WM-400 Cape Information`_ for more detail.
+
+System Clock
+============
+
+The SAM4L MCU is configured to use the 12 MHz internal oscillator on the board
+with the on-chip PLL to generate an 48 MHz system clock.
+
+Serial Port
+===========
+
+The ATSAM4LC4B MCU has 4 USARTs. One of the USARTs (USART3) is shared between
+RS-232 and RS-485 interfaces. The default console terminal is available at
+RS-232 onboard port or via USB device.
+
+Programming and Debugging
+*************************
+
+The SAM4L WM-400 Cape board needs a Segger `J-Link OB <https://www.segger.com/jlink-ob.html>`_.
+This provides a debug interface to the SAM4LC4B chip. You can use Ozone or
+JLink to communicate with the SAM4LC4B. The SAM4LC4B can be flashed using
+SAM-BA bootloader too.
+
+Flashing
+========
+
+#. Download JLink from the Segger `JLink Downloads Page`_. Go to the section
```

Comment:
isn't there a common doc page for jlink running?

Reply:
updated